docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -696,9 +697,123 @@ To read and write a value of the Bluetooth descriptor:
+<!-- TODO #07
+      - write about stop()
+ -->
+### Stop the server
+<!-- END #07 -->
+
+<!-- TODO #08
+      - write about addConnectStateChangeListener()
+      - write about removeConnectStateChangeListener()
+ -->
+### Receiving notifications on server's Connection State changes
+<!-- END #08 -->
+
+<!-- TODO #09
+      - write about getConnectionMtu()
+ -->
+### Accessing connection's ATT MTU
+<!-- END #09 -->
+
+<!-- TODO #10
+      - write about registerService()
+ -->
+### Registering services
+<!-- END #10 -->
+
+<!-- TODO #11
+      - write about unregisterAllServices()
+      - write about BluetoothGATTServerService::unregister()
+ -->
+### Unregistering services
+<!-- END #11 -->
+
+<!-- TODO #12
+      - write about setReadValueRequestCallback() for characteristic
+      - write about setWriteValueRequestCallback() for characteristic
+ -->
+### Managing characteristics of server service
+<!-- END #12 -->
+
+<!-- TODO #13
+      - write about setReadValueRequestCallback() for descriptor
+      - write about setWriteValueRequestCallback() for descriptor
+ -->
+### Managing descriptors of server service's characteristic
+<!-- END #13 -->
+
+### Managing the Advertising Options
+
+The Bluetooth Low Energy technology allows a device to broadcast some information without a connection between devices. The Bluetooth Low Energy API provides methods to control this advertising (broadcasting).
+
+To control what information is advertised by the device:
+
+1. Retrieve a `BluetoothLEAdapter` object (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothLEAdapter) and [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothLEAdapter) applications) with the `getLEAdapter()` method of the `BluetoothManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothManager) and [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothManager) applications):
+
+   ```
+   var adapter = tizen.bluetooth.getLEAdapter();
+   ```
+
+2. Set up options and start advertising with the `startAdvertise()` method of the `BluetoothLEAdapter` interface:
+
+   ```
+   var advertiseData = new tizen.BluetoothLEAdvertiseData({
+       includeName: true,
+       serviceuuids: ['180f'] /* 180F is 16bit Battery Service UUID */
+   });
+   var connectable = true;
+
+   adapter.startAdvertise(advertiseData, 'ADVERTISE', function onstate(state) {
+       console.log('Advertising configured: ' + state);
+   }, function(error) {
+       console.log('startAdvertise() failed: ' + error.message);
+   }, 'LOW_LATENCY', connectable);
+   ```
+
+   > **Note**
+   > To learn how to make your mobile device visible to other Bluetooth devices, see [Managing the Local Bluetooth Adapter](#managing-the-local-bluetooth-adapter).
+
+3. To disable the advertising, use the `stopAdvertise()` method of the `BluetoothLEAdapter` interface:
 
+   ```
+   adapter.stopAdvertise();
+   ```
+<!-- TODO #14
```

Comment:
As I've written at the top of the document - I think, these utilities and functions used to work with UUIDs should be put in the beginning of the document - they can be used for all APIs.